contrib/externs/angular-1.5.js

```diff
@@ -1795,6 +1795,20 @@ angular.$locale = function() {};
  */
 angular.$locale.prototype.id;
 
+/**
+ * @typedef {{
+ *     CURRENCY_SYM: (string),
+ *     GROUP_SEP: (string),
+ *     DECIMAL_SEP: (string)
+ * }}
+ */
+angular.$locale.NUMBER_FORMATS;
```

Comment:
Sorry I wasn't looking too closely. `@record` doesn't make sense here. Since this is an actual property available from Angular, what you had was almost right. I should have told you to just change `@typedef` to `@type`.